Tests/MapboxMapsTests/Style/StyleIntegrationTests.swift

```diff
@@ -276,4 +276,33 @@ internal class StyleIntegrationTests: MapViewIntegrationTestCase {
         XCTAssertTrue(sourceTerrainProperty is NSNull)
         XCTAssertTrue(exaggerationTerrainProperty is NSNull)
     }
+
+    func testCustomVectorSource() {
```

Comment:
Since the thing we're testing is `allSourceIdentifiers`, let's name the test to reflect that:

```suggestion
    func testAllSourceIdentifiersOmitsIdentifiersForCustomVectorSources() {
```